P9Operaters/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿Console.WriteLine("Give me a number of seconds");
+string? input = Console.ReadLine();
+float seconds = float.Parse(input);
+
+int iSeconds = (int)seconds % 60; //For print-out in seconds, without counting above
+float minutes = seconds / 60;
+
+int iMinutes = (int)minutes % 60; //For print-out in minutes, without counting above
+float hour = minutes / 60;
+
+int iHour = (int)hour % 24; //For print-out in hours, without counting above
+float days = hour / 24;
+
+int iDays = (int)days;
+
+Console.WriteLine($" Seconds: {iSeconds}\n Minutes: {iMinutes}\n Hour: {iHour}\n Days: {iDays}");
```

Comment:
Well done! Only thing I might criticize: we're not in 1990's C++ using the hungarian notation. We don't mark integers with `i` in their name. But I understand why you started it (due to the ambiguity of `seconds` and `iSeconds`, but maybe name them `totalSeconds` and `seconds` instead.)